pkg/functionconfig/types.go

```diff
@@ -77,6 +77,7 @@ type Trigger struct {
 	Annotations                           map[string]string `json:"annotations,omitempty"`
 	WorkerAvailabilityTimeoutMilliseconds *int              `json:"workerAvailabilityTimeoutMilliseconds,omitempty"`
 	WorkerAllocatorName                   string            `json:"workerAllocatorName,omitempty"`
+	ExplicitAckMode                       ExplicitAckMode   `json:"explicitAckMode,omitempty"`
```

Comment:
how about inferring from function's label as well?

Reply:
Which labels?
Are you maybe referring to the annotations (same as other kafka configurations)?

Reply:
yes

Reply:
Why put it in the annotations if it's already in the trigger spec?
Should be in only one place imo